src/Simplex/Chat/Migrations/chat_schema.sql

```diff
@@ -589,6 +590,23 @@ CREATE TABLE note_folders(
   unread_chat INTEGER NOT NULL DEFAULT 0
 );
 CREATE TABLE app_settings(app_settings TEXT NOT NULL);
+CREATE TABLE server_operators(
+  server_operator_id INTEGER PRIMARY KEY AUTOINCREMENT,
+  server_operator_tag TEXT,
+  trade_name TEXT NOT NULL,
+  legal_name TEXT,
+  enabled INTEGER NOT NULL DEFAULT 1,
+  role_storage INTEGER NOT NULL DEFAULT 1,
+  role_proxy INTEGER NOT NULL DEFAULT 1
+);
+CREATE TABLE usage_conditions_acceptance(
+  usage_conditions_acceptance_id INTEGER PRIMARY KEY AUTOINCREMENT,
+  server_operator_id INTEGER REFERENCES server_operators(server_operator_id),
+  conditions_required_at TEXT NOT NULL,
```

Comment:
we may need to set it in addtional api actually, because we don't know when conditions will be shown to user. so this would be nullable